### ISSUE TITLE
Clarify goals and non-goals

### ISSUE THREAD

**Issue by mwestphal** [f3d-app/f3d#1239]
F3D readme is pretty clear and descriptive, but it misses the goals and non-goals of the project,
lets clarify them.

Please note this is about F3D, not the libf3d, which may be slighlty different.

Resources:
 - https://blog.davidlyness.com/non-goals/

**Comment by mwestphal** [comment]
https://blog.davidlyness.com/non-goals/

**Comment by Meakk** [comment]
I think it summarizes it pretty well. I would just add:
- Support all industries dealing with 3D datasets

**Comment by mwestphal** [comment]
Sounds good to me, Ill add it!

**Comment by lknknm** [comment]
I have some Goal ideas that popped up in my head:
- Support any form of collaborative visualization — Being able to share visualization sessions or viewports is an example;
- Improve documentation for newcomers;
- Support all PBR maps without the need of raytracing (roughness, bump, occlusion, etc).
- Add some more features to manipulate lighting.

Let me know if you agree with any of these (:

**Comment by mwestphal** [comment]
Hey @lknknm 

Thanks for the inputs! This sounds more like feature ideas than goal/non-goals.
It is a bit different.

That being said, these are great feature ideas, you definitely should open issue for these. Do not hesitate to flesh them out a bit :)

**Comment by kathleenhang** [comment]
Great write-up! This really helps me to better understand what the app is meant to be. I have an idea:
- Be accessible for casual/disabled users

I really enjoy apps that can be used with just the trackpad and click of the mouse. I guess you already have that covered with "Be used non-interactively". Some examples are some video games I've played such as Civilization, RimWorld, and Balders Gate.

**Comment by mwestphal** [comment]
> Be accessible for casual users

TBH I'm not sure F3D is accessible to casual users. Most casual users are used to menus and buttons to cliks, which is not what F3D is. In order to be able to use F3D, you need to learn the bindings, which is not casual user friendly imo.

> Be accessible for disabled users

That I think **may** be in line with F3D, however we (the matainers at least) have literally **zero** experience with accessibility features. afaiu, here is a small list of what could be considered accessibility features:

 1. colorblind mode
 
 While we try to keep constrast correct, we never considered adding a default colormap for colorblind. We do have #228 and may add more colormaps in the future, but right now, I dont think we have that.
 
 2. low vision mode
 
 While we do not have a low vision mode, it is possible to set the font and font size, so at least there is some features there. We will not have a text-to-speech mode. It doesnt really makes sense for a 3D viewer.
 
 3. limited-mobility
 
 Happy with this one, cause we have shortcut for everything except a few interactions with the mouse (that can be controlled from the command line)
 
 In conclusion, I will create an issue for the colorbind mode (it was long coming anyway) but I'm not sure about stating a disability goal because of our lack of experience.
 
 > I really enjoy apps that can be used with just the trackpad and click of the mouse.
 
 I'm confused by this comment. F3D cannot be used only with trackpad, you need they keyboard.
